src/qt/forms/openuridialog.ui

```diff
@@ -30,6 +30,23 @@
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QPushButton" name="pasteButton">
+       <property name="toolTip">
+        <string>Paste address from clipboard</string>
+       </property>
+       <property name="text">
+        <string/>
+       </property>
+       <property name="icon">
+        <iconset resource="../bitcoin.qrc">
+         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
+        </property>
+        <property name="shortcut">
+         <string>Alt+P</string>
```

Comment:
not sure how I feel about this shortcut. How intuitive/practical is this?

Reply:
https://github.com/bitcoin/bitcoin/pull/17955#discussion_r368223931
https://github.com/bitcoin/bitcoin/pull/17955#discussion_r368906725

Reply:
NACK the shortcut, see #17955 review.